admin/assets/javascripts/discourse/helpers/commit-url.js

```diff
@@ -0,0 +1,19 @@
+import { helper as buildHelper } from "@ember/component/helper";
+import { htmlSafe } from "@ember/template";
+
+export default buildHelper(function (params) {
```

Comment:
I don't think in this case you need `buildHelper`. I think you could do:

```javascript
export default function commitUrl(cssClass, version, prettyVersion, url) {
}
```

Then you can get rid of your `const` line below too